test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
```

Comment:
I was thinking about this, but I think I want to add tests to make sure it throws errors when there is no controllers directory etc..